activiti-cloud-services-org/activiti-cloud-services-org-jpa/src/main/java/org/activiti/cloud/services/organization/jpa/GroupRepository.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.organization.jpa;
+
+import org.activiti.cloud.organization.core.model.Group;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+/**
+ * JPA Repository for {@link Group} entity
+ */
+@RepositoryRestResource(path = "groups",
+        collectionResourceRel = "groups",
+        itemResourceRel = "groups")
+public interface GroupRepository extends JpaRepository<Group, String> {
```

Comment:
I've choose the `JpaRepository` because extends the `PageAndSortingRepository` and was used also in the official documentation for the Spring Data Rest but I can switch to `PageAndSortingRepository`.

Reply:
My mistake, you're right JpaRepository is already extending PageAndSortingRepository. I tough it was not.